CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Add PostLogger support to MeetingProvider
+- [Demo] Add bandwidth metric display
```

Comment:
Also addition of hook to library? Will help in knowing what updated in the library as well. This may confuse to be only demo addition.